lnrpc/lightning.proto

```diff
@@ -2755,13 +2755,6 @@ message Hop {
     */
     string pub_key = 8;
 
-    /*
-    If set to true, then this hop will be encoded using the new variable length
-    TLV format. Note that if any custom tlv_records below are specified, then
-    this field MUST be set to true for them to be encoded properly.
-    */
-    bool tlv_payload = 9;
```

Comment:
Set this to `reserved` instead and add a comment that it's a removed field and that the number shouldn't be re-used?